### ISSUE TITLE
[BUG]  try to run an example 

### ISSUE THREAD

**Issue by thanaphon** [JoMingyu/google-play-scraper#66]
**google_play_scraper.VERSION**
Write result of `google_play_scraper.VERSION`

**Describe the bug**
A description of what the bug is.

Try to run an example code, but got an error on the app() method, but the review() method works fine.  

Ps. Thank you for a great module. 

**Code**
Copy and paste the code that have issue.

```

from google_play_scraper import app
txt = app('com.ubisoft.might.magic.heroes.fantasy.rpg')

---------------------------------------------------------------------------
JSONDecodeError                           Traceback (most recent call last)
<ipython-input-53-1296b0fc4cb0> in <module>()
      1 from google_play_scraper import app
----> 2 txt = app('com.ubisoft.might.magic.heroes.fantasy.rpg')

2 frames
/usr/lib/python3.6/json/decoder.py in decode(self, s, _w)
    340         end = _w(s, end).end()
    341         if end != len(s):
--> 342             raise JSONDecodeError("Extra data", s, end)
    343         return obj
    344 

JSONDecodeError: Extra data: line 202 column 1 (char 6038)


```

**Expected behavior**
A description of what you expected to happen.

**Additional context**

**Comment by JoMingyu** [comment]
Google Play restructured their dataset. I'll fix it.

**Comment by JoMingyu** [comment]
It fixed at version 0.1.1. Check it out!
